easybuild/easyconfigs/s/Spyder/Spyder-6.0.1-gfbf-2023b.eb

```diff
@@ -0,0 +1,118 @@
+easyblock = 'PythonBundle'
+
+name = 'Spyder'
+version = '6.0.1'
+
+homepage = 'https://github.com/spyder-ide/spyder'
+description = """Spyder is an interactive Python development environment providing MATLAB-like features in a simple
+ and light-weighted software."""
+
+toolchain = {'name': 'gfbf', 'version': '2023b'}
+
+builddependencies = [
+    ('poetry', '1.6.1'),
+    ('maturin', '1.3.1'),
+]
+dependencies = [
+    ('Python', '3.11.5'),
+    ('Python-bundle-PyPI', '2023.10'),
+    ('aiohttp', '3.9.5'),
+    ('IPython', '8.17.2'),
+    ('jedi', '0.19.1'),
+    ('jupyter-server', '2.14.0'),
+    ('Pylint', '3.2.5'),
+    ('PyZMQ', '25.1.2'),
+    ('PyQt5', '5.15.10'),
+    ('Qtconsole', '5.6.0'),
+    ('Rtree', '1.2.0'),
+    ('Deprecated', '1.2.14'),
+    ('tornado', '6.4'),
```

Comment:
What is tornado used for? Haven't found it to be (strictly) required, i.e. `pip check` doesn't complain

Reply:
I found it in optional dependencies.